pkg/codegen/go/gen.go

```diff
@@ -809,9 +840,19 @@ func (pkg *pkgContext) genHeader(w io.Writer, goImports []string, importedPackag
 	var imports []string
 	if len(importedPackages) > 0 {
 		for k := range importedPackages {
-			imports = append(imports, k)
+			imports = append(imports, fmt.Sprintf("%s", k))
 		}
 		sort.Strings(imports)
+
+		// Match the end of an import path that can be mapped to an alias.
+		//   "github.com/pulumi/pulumi-kubernetes/sdk/go/kubernetes/meta/v1" -> meta/v1
+		re := regexp.MustCompile(`\w+/\w+$`)
```

Comment:
Instead of this, we should normalize the module -> alias map to a package import -> alias map. That will avoid the need for this kind of matching, which won't work for packages that don't match this pattern.